Diary/Controller/DiaryViewController.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  Diary - DiaryViewController.swift
+//  Created by yagom. 
+//  Copyright © yagom. All rights reserved.
+//  last modified by Mary & Whales
+
+import UIKit
+
+final class DiaryViewController: UIViewController {
+    private let diaryManager = DiaryManager()
+    
+    private var tableView: UITableView = {
+        let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return tableView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureView()
+        configureTableView()
+        setUpConstraints()
+    }
+    
+    private func configureView() {
+        view.backgroundColor = .systemBackground
+        view.addSubview(tableView)
+        
+        configureNavigationItem()
+    }
+    
+    private func configureNavigationItem() {
+        let addDiaryBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add,
```

Comment:
버튼이 눌렸을 때 실행되는 함수의 이름은
`실행하는 행위` 보다는 `일어난 일`을 표현해주는게 통상적으로 좋을거 같아요 ㅎㅎ
왜냐하면, 나중에 객체가 분리되면, 뷰와 뷰모델이 분리되는데, 뷰에서 일어난 일을 뷰모델한테 전달만 해주고,
뷰모델이 어떤일을 할지 결정하거든요 ㅎㅎ
```suggestion
       action: #selector(addDiaryButtonTapped))
```

Reply:
관련 커밋: a02791e21400ec4c4af5dc690bfd127eda894f68

해당 메서드명 수정하였습니다! 🙇‍♀️